src/styles.css

```diff
@@ -1577,63 +1577,66 @@ aside {
-      text-decoration: underline;
+
+.search-pager__item.is-active .search-pager__item-link {
+  font-size: 1em;
+  line-height: 1.46667em; }
+  @media (min-width: 450px) {
+    .search-pager__item.is-active .search-pager__item-link {
+      background-color: #737373;
+      color: #fff; }
+      .search-pager__item.is-active .search-pager__item-link:hover, .search-pager__item.is-active .search-pager__item-link:active {
+        background-color: black; } }
+
+.search-pager__item.not-active .search-pager__item-link {
+  display: none; }
+  @media (min-width: 450px) {
+    .search-pager__item.not-active .search-pager__item-link {
+      display: inline-block; } }
+
+.search-pager__item-link {
+  transition: all 0.3s ease;
+  font-size: 0.9em;
+  line-height: 1.62963em;
+  font-weight: 700;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 0;
+  text-align: center;
+  padding: 0.36667em; }
+  @media (min-width: 450px) {
+    .search-pager__item-link {
+      background-color: #0000ff;
       color: #fff;
-      background-color: #0000ff; }
-      @media (min-width: 450px) {
-        .search-pager__item a:hover, .search-pager__item a:active {
-          font-size: 1em;
-          line-height: 1.46667em;
-          background-color: navy; } }
-      .search-pager__item a:hover svg, .search-pager__item a:active svg {
-        fill: black; }
-    .search-pager__item a svg {
-      height: 1.3em;
-      fill: #fff;
-      display: block; }
-      @media (min-width: 450px) {
-        .search-pager__item a svg {
-          height: 1em; } }
-  .search-pager__item.is-active a {
-    font-size: 1em;
-    line-height: 1.46667em; }
+      border-radius: 3px;
+      width: 2.2em;
+      height: 2.2em;
+      padding: 0; } }
+  @media (min-width: 900px) {
+    .search-pager__item-link {
+      font-size: 0.8em;
+      line-height: 1.65em;
+      width: 1.76em;
+      height: 1.76em; } }
+  .search-pager__item-link:hover, .search-pager__item-link:active {
+    text-decoration: underline;
+    color: #fff;
+    background-color: #0000ff; }
     @media (min-width: 450px) {
-      .search-pager__item.is-active a {
-        background-color: #737373;
-        color: #fff; }
-        .search-pager__item.is-active a:hover, .search-pager__item.is-active a:active {
-          background-color: black; } }
-  .search-pager__item.not-active {
-    display: none; }
+      .search-pager__item-link:hover, .search-pager__item-link:active {
+        font-size: 1em;
+        line-height: 1.46667em;
+        background-color: navy; } }
+    .search-pager__item-link:hover svg, .search-pager__item-link:active svg {
+      fill: black; }
+  .search-pager__item-link svg {
+    height: 1.3em;
+    fill: #fff;
+    display: block; }
     @media (min-width: 450px) {
-      .search-pager__item.not-active {
-        display: inline-block; } }
+      .search-pager__item-link svg {
+        height: 1em; } }
```

Comment:
@nstriedinger can you walk me through this change too? Does this just happen to look like a change in the css rule because of how things were rearranged or is an actual change?